spec/v1/digest_set.md

```diff
@@ -1,27 +1,29 @@
 # DigestSet field type specification
 
-Version: v1.0
+Version: v1.1
 
-Set of one or more cryptographic digests for a single software artifact or
-metadata object.
+Set of one or more immutable digests for a single software artifact or metadata
+object.
```

Comment:
nit: a digest is always immutable since it's just a string. I think "immutable reference" is more accurate. Also I think it may be worth calling out that the overwhelming majority use case is cryptographic digest, to avoid confusion.

```suggestion
Set of one or more cryptographic digests, or other immutable references, for a
single software artifact or metadata object.
```

(Also ok to just say "immutable references" - your choice)

Reply:
Done